pallav/fashionTech/pallavBanka/parkingAssignment/Services/ParkingCharge.java

```diff
@@ -0,0 +1,23 @@
+package fashionTech.pallavBanka.parkingAssignment;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ParkingCharge {
+    private Long ParkingCharges(VehicleConstant.VehicleType vehicleType, Date inTime, Date outTime) {
+        Map<VehicleConstant.VehicleType, Integer> charges = new HashMap();
+        charges.put(VehicleConstant.VehicleType.twowheeler, 20);
+        charges.put(VehicleConstant.VehicleType.fourwheeler, 40);
+        charges.put(VehicleConstant.VehicleType.sixwheeler, 80);
+        if (outTime != null){
+            //difference in time in milliseconds
+            Long diff = outTime.getTime() - inTime.getTime();
+            Long diffHours = diff / (3600000) % 24;
```

Comment:
You are making a division here and returning 'Long', you should handle how you are going to round it off, whether Ceil or Floor.

Reply:
You are taking remainder with 24, is it perDay charges... the variable should specify wether charges are perDay or perHour.